docs/source/markdown/podman-create.1.md.in

```diff
@@ -491,8 +491,9 @@ In order for users to run rootless, there must be an entry for their username in
 
 Rootless Podman works better if the fuse-overlayfs and slirp4netns packages are installed.
 The fuse-overlayfs package provides a userspace overlay storage driver, otherwise users need to use
-the vfs storage driver, which is diskspace expensive and does not perform well. slirp4netns is
-required for VPN, without it containers need to be run with the --network=host flag.
+the vfs storage driver, which is diskspace expensive and does not perform well.
```

Comment:
```suggestion
the vfs storage driver, which can be disk space expensive and less performant than other drivers.
```

Reply:
I know it's mostly pre-existing, but please change.

Reply:
Done.